package.json

```diff
@@ -86,4 +86,4 @@
   "msw": {
     "workerDirectory": "public"
   }
-}
+}
```

Comment:
this file should not change please check out why is it changing as there is no difference in old and new version

Reply:
am not sure , i made some some changes in package.json and rolled it back to original values, as you can see there are no other changes, i think maybe it's because of liniting software

Reply:
yes, that is because of linting software so you need to fix it. it can not be left unchecked as it may cause same linting issues in someone else's system. So fix it

Reply:
Check out other PR's why arent these changes reflected in them?

Reply:
okay will check, thanks

Reply:
fixed